be/src/http/action/download_action.cpp

```diff
@@ -68,7 +69,7 @@ void DownloadAction::handle_normal(HttpRequest* req, const std::string& file_par
         status = check_token(req);
         if (!status.ok()) {
             std::string error_msg = status.to_string();
-            if (status.is_not_authorized()) {
```

Comment:
we should remove these method:
    bool is_invalid_argument() const { return ErrorCode::INVALID_ARGUMENT == _code; }

    bool is_not_found() const { return _code == ErrorCode::NOT_FOUND; }
    bool is_not_authorized() const { return code() == TStatusCode::NOT_AUTHORIZED; }

I think only IO ERROR should has method is_io_error.